### PR TITLE
Update docker build

### PR DESCRIPTION
**Pull request by markjezreeldagot**
## Description
Update docker build since the updated module fails on python3.5

## Screenshot
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM python:alpine3.15 AS base
+FROM python:3.8.5-slim-buster AS base
```
Comment (gengo-it) [inline Dockerfile:1]: why is the OS changed?

Comment (markjezreeldagot) [reply Dockerfile:1]: the updated Modules dont work

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 MAINTAINER Gengo Dev Team
 RUN apk add build-base linux-headers pcre-dev
```
Comment (shinji-kohara) [inline Dockerfile]: Is the base image `python:3.8.5-slim-buster` still alpine? If so, it's okay but I think usually debian based OS is used so don't we need to replace `apk` command to `apt`?

Comment (mervinsanandres) [reply Dockerfile]: The [base image](https://github.com/docker-library/python/blob/06e0db2fc2e32eb95771b23bc8ad36bf9450640a/3.8/slim-buster/Dockerfile) looks like Debian so `apt-get` might be needed

**Comment by markjezreeldagot** [comment]
Im not really good at building Dockerfile but i have this error so i cant build this application
https://ci.gengo.com/job/Pipelines/job/geoip-lite-api/1/console

File: Dockerfile [hunk 3]
```diff
@@ -1,11 +1,20 @@
 FROM python:3.8.5-slim-buster AS base
 MAINTAINER Gengo Dev Team
-RUN apk add build-base linux-headers pcre-dev
-RUN pip install uwsgi==2.0.21 && pip install awscli
+
+ARG uid=1000
+
+RUN apt-get -y update
```
Comment (mervinsanandres) [inline Dockerfile]:
```suggestion
RUN apt-get -y update && apt-get -y install build-essential
```

**Review by shinji-kohara — APPROVED** [review]

**Review by mervinsanandres — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -1,11 +1,20 @@
...
+RUN apt-get -y update && apt-get -y install build-essential
+
+RUN pip install --upgrade pip
+
+RUN adduser -u ${uid} --disabled-password --disabled-login --gecos python python
+USER python
+
+RUN pip install --no-warn-script-location uwsgi==2.0.21 awscli
 
 FROM base as app
```
Comment (gengo-it) [inline Dockerfile:15]: neither `base` nor `app` is used
With this line, `apt-get`, `adduser`, etc. above are all meaningless.

Comment (gengo-it) [reply Dockerfile:15]: oh, maybe I'm wrong.
It's not meaningless, but there is no need to have 2 `FROM`s

Comment (gengo-it) [reply Dockerfile:15]: `uwsgi` and `awscli` are installed with `requirements.txt`.
This line should be deleted.
```
RUN pip install --no-warn-script-location uwsgi==2.0.21 awscli
```

Comment (markjezreeldagot) [reply Dockerfile:15]: Adduser is needed for the Python Script Warning 
Apt-get update also need for build-essentials

Comment (markjezreeldagot) [reply Dockerfile:15]: https://github.com/gengo/geoip-lite-api/pull/6 all of the changes that are requested are here @ninotoshi

**Pull request merged**